web/vtadmin/src/index.css

```diff
@@ -51,8 +54,8 @@
 
     /* Inputs + other form controls */
     --inputHeightSmall: 2.4rem;
-    --inputHeightMedium: 3.6rem;
-    --inputHeightLarge: 4.8rem;
+    --inputHeightMedium: 3.7rem;
+    --inputHeightLarge: 4.6rem;
```

Comment:
These adjustments are a gross little hack to get text inputs + buttons to line up vertically, since the inputs don't use tailwind util classes (yet) and are therefore off by a few pixels. Will revisit the inputs later!